scripts/integration/clickhouse/compose.yaml

```diff
@@ -2,4 +2,4 @@ version: '3'
 
 services:
   clickhouse:
-    image: docker.io/yandex/clickhouse-server:${CONFIG_VERSION}
+    image: docker.io/clickhouse/clickhouse-server:${CONFIG_VERSION}
```

Comment:
The `yandex/clickhouse-server` image is no longer maintained.

Reply:
NP, the change you made rang a bell and I remembered that PR never got closed out/completed.